react/src/App.js

```diff
@@ -2,17 +2,22 @@ import { Route, Routes } from 'react-router-dom';
 import HomePage from './pages/Home';
 import Menu2Page from './pages/Menu2';
 import Menu3Page from './pages/Menu3';
-import PogodaPage from './pages/Pogoda';
+import WeatherPage from './pages/WeatherPage';
+import WeatherSearchPage from './pages/WeatherSearchPage';
+import WeatherToday from './pages/WeatherToday';
+import WeatherTermPage from './pages/WeatherTermPage';
 import './App.css';
 
-
-function App() {
+const App = () => {
   return (
-    <Routes> 
-      <Route path='/' element={<HomePage/>} /> 
-      <Route path='/menu2' element={<Menu2Page/>} /> 
-      <Route path='/menu3' element={<Menu3Page/>} /> 
-      <Route path='/pogoda' element={<PogodaPage/>} /> 
+    <Routes>
+      <Route path='/' element={<HomePage />} />
+      <Route path='/menu2' element={<Menu2Page />} />
+      <Route path='/menu3' element={<Menu3Page />} />
+      <Route path='/pogoda' element={<WeatherPage />} />
+      <Route path='/pogoda/szukaj' element={<WeatherSearchPage />} />
+      <Route path='/pogoda/dzis' element={<WeatherToday />} />
+      <Route path='/pogoda/term' element={<WeatherTermPage />} />
```

Comment:
/pogoda/term' - takie rzeczy fajnie by było trzymać w jakimś pliku, coś ala ten plik z api bo w innym np.: ButtonsWeather znowu to używasz